deploy/bz1997062/config.yaml

```diff
@@ -0,0 +1,3 @@
+deploymentMode: "Direct"
+direct:
+    environments: ["production", "stage", "integration"]
```

Comment:
I don't think this is what you intend. You only want this to go to hive clusters?